doc/RelNotes.md

```diff
@@ -20,13 +20,14 @@ if you find any other issue with this release.
 
 # Changes {#Changes}
 
-## git master (0.6)
+## Deflect 0.6
 
 * Many classes of the Server part have been renamed
 * Various API fixes
 * Project-wide codestyle cleanup
 * DesktopStreamer: Minor fixes for desktop interaction, only show checkbox on
   supported platforms (OSX)
+* DesktopStreamer OSX: zip the application for Puppet deployment
```

Comment:
this is part of the (unreleased) 0.6.1. Maybe put in under a new section, e.g. git-master like in https://github.com/Eyescale/Equalizer/pull/463/files#diff-f04a81819f7614362e00f5804d9ceb0a?

Besides, you want to add the release dates as well to the changelog?